src/ifd-nfc.c

```diff
@@ -444,8 +519,12 @@ IFDHGetCapabilities(DWORD Lun, DWORD Tag, PDWORD Length, PUCHAR Value)
       *Value  = 1;
       *Length = 1;
       break;
+#if defined(HAVE_DECL_TAG_IFD_STOP_POLLING_THREAD) && HAVE_DECL_TAG_IFD_POLLING_THREAD
```

Comment:
I think you're mixing up `HAVE_DECL_TAG_IFD_STOP_POLLING_THREAD` with `TAG_IFD_STOP_POLLING_THREAD`. The former will (or will not be) defined in `config.h` if the latter is available (`1`) or not available (`0`). That's why the `#if ...` always uses `HAVE_DECL_TAG_IFD_STOP_POLLING_THREAD` instead of `TAG_IFD_STOP_POLLING_THREAD`.